### PR TITLE
Add section on static IP addresses

### PR DESCRIPTION
**Pull request by kr8n3r**
What
----

Signpost links to paas-admin pages where ip addresses are viewable

Why here? because users search our documentation, but we don't want them to be easily publicly available.

How to review
-------------

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.

Who can review
--------------

not @kr8n3r

### CODE REVIEW

**Comment by pauldougan** [comment]
@kr8n3r The other page in our docs that mentions egress ip is https://docs.cloud.service.gov.uk/monitoring_apps.html#configure-app

this page advises the user to contact the support team, I think we should point them to the new content in the guidance section for consistency.

**Comment by kr8n3r** [comment]
> @kr8n3r The other page in our docs that mentions egress ip is https://docs.cloud.service.gov.uk/monitoring_apps.html#configure-app
> 
> this page advises the user to contact the support team, I think we should point them to the new content in the guidance section for consistency.

added in https://github.com/alphagov/paas-tech-docs/pull/405/commits/c4b0732ffb503ed3529a06346278d732d80a12f1

File: source/documentation/guidance/static-egress-ips.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Static egress IP addresses
+
+In order to locate the GOV.UK PaaS static egress Internet Protocol (IP) addresses please [log in to your GOV.UK PaaS account](https://www.cloud.service.gov.uk/sign-in/) and go to the relevant link (depending on where your account is located):
```
Comment (cmcnallygds) [inline source/documentation/guidance/static-egress-ips.md:3]:
```suggestion
To locate the GOV.UK PaaS static egress Internet Protocol (IP) addresses, [log in to your GOV.UK PaaS account](https://www.cloud.service.gov.uk/sign-in/) and go to the relevant link (depending on where your account is located):
```

File: source/documentation/guidance/static-egress-ips.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+- [London static IP addresses](https://admin.london.cloud.service.gov.uk/support/static-ip)
```
Comment (cmcnallygds) [inline source/documentation/guidance/static-egress-ips.md:5]: Should we say 'static egress' to be consistent?

**Review by cmcnallygds — COMMENTED** [review]
Made a few suggestions about terminology

File: source/documentation/guidance/static-egress-ips.md (hunk 2 above)
Comment (kr8n3r) [reply source/documentation/guidance/static-egress-ips.md]: updated

**Comment by markpeterbuckley** [comment]
LGTM

**Pull request merged**